src/main/java/com/aws/greengrass/util/platforms/unix/UnixPlatform.java

```diff
@@ -505,44 +516,44 @@ protected FileSystemPermissionView getFileSystemPermissionView(FileSystemPermiss
         if (permissionView instanceof PosixFileSystemPermissionView) {
-            PosixFileSystemPermissionView posixFileSystemPermissionView =
-                    (PosixFileSystemPermissionView) permissionView;
+            PosixFileSystemPermissionView posixFileSystemPermissionView = (PosixFileSystemPermissionView) permissionView;
             Set<PosixFilePermission> permissions = posixFileSystemPermissionView.getPosixFilePermissions();
 
             PosixFileAttributeView view = Files.getFileAttributeView(path, PosixFileAttributeView.class,
                     LinkOption.NOFOLLOW_LINKS);
 
             Set<PosixFilePermission> currentPermission = view.readAttributes().permissions();
             if (!currentPermission.equals(permissions)) {
-                logger.atTrace().setEventType(SET_PERMISSIONS_EVENT).kv(PATH_LOG_KEY, path).kv("perm",
-                        PosixFilePermissions.toString(permissions)).log();
+                logger.atTrace().setEventType(SET_PERMISSIONS_EVENT).kv(PATH_LOG_KEY, path)
+                        .kv("perm", PosixFilePermissions.toString(permissions)).log();
                 view.setPermissions(permissions);
             }
         }
     }
 
     /**
      * Run a arbitrary command.
-     * @param cmdStr command string
-     * @param out output consumer
-     * @param msg error string
-     * @throws IOException IO exception
+     *
+     * @param cmdStr
+     *            command string
+     * @param out
+     *            output consumer
+     * @param msg
+     *            error string
+     * @throws IOException
+     *             IO exception
      */
-    public void runCmd(String cmdStr, Consumer<CharSequence> out, String msg)
-            throws IOException {
+    public void runCmd(String cmdStr, Consumer<CharSequence> out, String msg) throws IOException {
         try (Exec exec = getInstance().createNewProcessRunner()) {
             StringBuilder output = new StringBuilder();
             StringBuilder error = new StringBuilder();
-            Optional<Integer> exit = exec.withExec(cmdStr.split(" "))
-                    .withShell()
-                    .withOut(o -> {
-                        out.accept(o);
-                        output.append(o);
-                    }).withErr(error::append).exec();
+            Optional<Integer> exit = exec.withExec(cmdStr.split(" ")).withShell().withOut(o -> {
+                out.accept(o);
+                output.append(o);
+            }).withErr(error::append).exec();
             if (!exit.isPresent() || exit.get() != 0) {
-                throw new IOException(String.format(
-                        String.format("%s - command: %s, output: %s , error: %s ", msg, cmdStr, output.toString(),
-                                error.toString())));
+                throw new IOException(String.format(String.format("%s - command: %s, output: %s , error: %s ", msg,
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Potential log injection detected. Ensure all untrusted input is properly sanitized before logging. Use parameterized logging or validate input against a allow list to prevent log injection vulnerabilities. Consider using a dedicated logging library&#39;s built-in sanitization features when available. Learn more - https://cwe.mitre.org/data/definitions/117.html